src/sentry/notifications/notification_action/utils.py

```diff
@@ -14,10 +14,21 @@
 logger = logging.getLogger(__name__)
 
 
-def should_fire_workflow_actions(org: Organization) -> bool:
-    return features.has(
-        "organizations:workflow-engine-single-process-workflows", org
-    ) or features.has("organizations:workflow-engine-trigger-actions", org)
+def should_fire_workflow_actions(org: Organization, type_id: int) -> bool:
+    ga_type_ids = options.get("workflow_engine.issue_alert.group.type_id.ga")
+    rollout_type_ids = options.get("workflow_engine.issue_alert.group.type_id.rollout")
+
+    return (
+        type_id in ga_type_ids  # We have completely rolled out these group types
+        or (
+            type_id
+            in rollout_type_ids  # While we are rolling out these groups & we are single  processing
+            and features.has("organizations:workflow-engine-single-process-workflows", org)
+        )
+        or features.has(
+            "organizations:workflow-engine-trigger-actions", org
+        )  # This is for temporary rollouts
+    )
```

Comment:
For anybody reading, this function looks like this because rules currently process all group types and can be gated by a single flag (`organizations:workflow-engine-single-process-workflows`). Therefore we can use a list to say which group types should actually go through workflow engine

When rolling out to a new group type, you can ramp up orgs on the flag and include it in the rollout list, and then move the group type to the GA list and reuse the flag to roll out for another group type